ci.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Copyright (c) 2016, Isaac I. Y. Saito
+# Copyright (c) 2017, Mathias Lüdtke
+# Copyright (c) 2017, Iñigo Martínez
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file remains as "travis.sh" at the top directory of industrial_ci repository only to keep backward compatibility between version 0.2.2 and the newer.
```

Comment:
Take my comment back. In fact we don't need this entire file since this is a new file.